johnsonandjohnson/Bodiless-JS
# feat(migration_tool): Migration tool enhancement to auto turn off Bodiless-Tailwind Theme

## Changes
- New `setupEnvVars` step for the migration tool which currently disables `BODILESS_TAILWIND_THEME_ENABLED` before compilation.
- New optional `disableTailwind ` param for `migration-settings.json` which controles `BODILESS_TAILWIND_THEME_ENABLED`. Set to `true` by default.

## Test Instructions
- Run `npm run sites:clone-local starter`
- Run `npm run sites:update starter`
- Go to `cd sites/starter`
- Make sure `BODILESS_TAILWIND_THEME_ENABLED` is commented out in `.env.site` by default
- Run `npm run migrate`
- Make sure `BODILESS_TAILWIND_THEME_ENABLED` is now set to `0` and not commented out in `.env.site`.

## Related Issues
Closes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/124
